src/webapp/gradebookServer.js

```diff
@@ -190,12 +190,25 @@ app.get('/seasons', function(request, response) {
       passwordText, request.query.host, request.query.port);
 
    //Get the params from the url
-   var instructorID = request.query.instructorid;
+   var userID = request.query.userid;
+   var userRole = request.query.userRole;
    var year = request.query.year;
-
-   //Set the query text
-   var queryText = 'SELECT SeasonOrder, SeasonName FROM gradebook.getInstructorSeasons($1, $2);';
-   var queryParams = [instructorID, year];
+   var queryText;
+   var queryParams;
+
+   //Set the query
+   var queryParams = [userID, year];
```

Comment:
```suggestion
   var queryParams;
```

Reply:
I see that makes sense. Thanks again for the changes.